test/unit/storage/utils/filters.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable mocha/no-pending-tests */
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const filterTypes = require('../../../../storage/utils/filter_types');
+const inputSerializers = require('../../../../storage/utils/inputSerializers');
+const filters = require('../../../../storage/utils/filters');
+
+const customSerializer = sinonSandbox.stub().returns('custom serialized value');
+
+describe('filters', () => {
+	beforeEach(done => {
```

Comment:
No need for done here, instead u can do
```
beforeEach(async () => {
  xxx
});
```

Reply:
I noticed that Elements is using `return Promise.resolve();`
We should be consistent on where to use `done()`, `return` and `async`
Let's discuss this and make the changes in another issue.